mailet/icalendar/src/main/java/org/apache/james/transport/mailets/ICalendarParser.java

```diff
@@ -134,10 +136,28 @@ public String getMailetInfo() {
             ByteArrayInputStream inputStream = new ByteArrayInputStream(icsContent);
             return Stream.of(Pair.of(key, builder.build(inputStream)));
         } catch (IOException e) {
-            LOGGER.error("Error while reading input: " + icsContent, e);
+            if (LOGGER.isErrorEnabled()) {
+                String icsContentAsString;
+                try {
+                    icsContentAsString = new String(icsContent, Charset.forName("UTF-8"));
+                } catch (Throwable t) {
```

Comment:
s/Throwable/Exception/ ?

Reply:
Fixed already (I did that remark to the contributor...)